--- a/jampal/src/META-INF/MANIFEST.MF
+++ b/jampal/src/META-INF/MANIFEST.MF
@@ -1,2 +1,2 @@
 Main-Class: pgbennett.jampal.Jampal
-Class-Path: /usr/share/java/freetts.jar /usr/share/java/looks.jar /usr/share/java/squareness.jar ../freetts/lib/freetts.jar liquidlnf.jar looks-2.3.0.jar napkinlaf-1.2.jar squareness.jar InfoNodeilf-gpl.jar lipstikLF-1.1.jar nimrodlf.j16.jar PgsLookAndFeel.jar
+Class-Path: /usr/share/java/looks.jar /usr/share/java/squareness.jar liquidlnf.jar looks-2.3.0.jar napkinlaf-1.2.jar squareness.jar InfoNodeilf-gpl.jar lipstikLF-1.1.jar nimrodlf.j16.jar PgsLookAndFeel.jar
